kuma/authkeys/decorators.py

```diff
@@ -9,11 +9,12 @@
 
 
 def accepts_auth_key(func):
-    """Enable a view to accept an auth key via HTTP Basic Auth.
+    """
+    Enable a view to accept an auth key via HTTP Basic Auth.
```

Comment:
I think we'll need a style guide, because I'm not sure of the rules.  I've been following PEP 257, which allows one-line docstrings on one line, but you seem to correct them to three lines.  And here, I'm used to PEP 257's example of an empty line between the one-line docstring and the elaborate description.  However, I'm not sure what standard you're aiming for, so I feel I can't judge if you are moving toward the standard or not.